docs/_ext/exampleinclude.py

```diff
@@ -0,0 +1,47 @@
+from docutils import nodes
+from docutils.parsers.rst import directives
+from sphinx.directives.code import LiteralIncludeReader
+from sphinx.util.docutils import SphinxDirective
+from sphinx.util.nodes import set_source_info
+
+
+class ExampleInclude(SphinxDirective):
```

Comment:
What are the benefits we get with this custom extension as compared to the Sphinx `literalinclude` directive?

Reply:
Example:

```
.. literalinclude:: example.txt
   :start-after: # START
   :end-before: # END
```

Reply:
I'll address it to make the link clickable in separate PR tracking it here https://github.com/astronomer/astronomer-providers/issues/484